main.py

```diff
@@ -0,0 +1,46 @@
+from transformers import pipeline
+import streamlit as st
+
+
+def get_text():
+    return st.text_area(
+        label="Введите текст-маску на английском или французском языках (желаемые для подбора слова замените на *? ):",
+        value="Diplomacy is the *? of telling *? to go to *? in such a way that they *?."
+    )
+
+
+@st.cache
+def unmask(text):
+    split_text = text.split("*?")
+    result_text = split_text[0]
+    results = [""]
+    for i in range(1, len(split_text)):
+        unmasker = pipeline('fill-mask', model='camembert-base')
```

Comment:
This will reload the pipeline for each sample. This is not an efficient way to work with neural networks. You need to load the pipeline only once and then use it for each unmasking.